src/revalue/utils.py

```
@@ -0,0 +1,165 @@
+    """
+    Dynamically imports and returns an ASReview component class.
+    component_type: e.g., 'classifiers', 'queriers'
+    component_name: e.g., 'NaiveBayes', 'Max' (as defined in YAML)
+    """
+    try:
+        class_path = ASREVIEW_COMPONENT_MAPPING[component_type][component_name]
+        module_path, class_name = class_path.rsplit('.', 1)
+        module = importlib.import_module(module_path)
+        return getattr(module, class_name)
+    except KeyError:
+        raise ValueError(f"Component type '{component_type}' or name '{component_name}' not found in mapping.")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)